services-gateway-client-transport/src/main/java/io/scalecube/services/gateway/transport/websocket/WebsocketGatewayClient.java

```diff
@@ -57,7 +59,11 @@ public WebsocketGatewayClient(GatewayClientSettings settings, GatewayClientCodec
                   if (settings.sslProvider() != null) {
                     tcpClient = tcpClient.secure(settings.sslProvider());
                   }
-                  return tcpClient.runOn(loopResources).host(settings.host()).port(settings.port());
+                  return tcpClient
+                      .wiretap(true)
```

Comment:
Maybe it is better to add an ability to configure it from the client's code using `GatewayClientSettings`.